### PR TITLE
feat: create User list page FE

### PR DESCRIPTION
**Pull request by nadiia-tsytsylina**
## Describe your changes

- created User List component;
- created Account details [id] page;
- updated global constants
- updated User type;
- added colors for admin pages.

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/CC-234)

image

image

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [ ] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [ ] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

**Comment by ProggerOleg** [comment]
Maybe let's try to make it in our app's design style, at least use Primary.main for icons/filter button. Also I think Users should be green, not purple

File: src/components/user-list/mocks.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+export const users = [
```
Comment (ProggerOleg) [inline src/components/user-list/mocks.ts:1]: constants should be uppercase

Comment (nadiia-tsytsylina) [reply src/components/user-list/mocks.ts:1]: it's mocked data, I will delete it after backend is ready

Comment (nadiia-tsytsylina) [reply src/components/user-list/mocks.ts:1]: Fixed colors

**Review by fedasevich — APPROVED** [review]

**Review by Simpsonkk — APPROVED** [review]

**Review by ProggerOleg — APPROVED** [review]

**Pull request merged**